src/controllers/AbstractController.ts

```diff
@@ -40,6 +44,14 @@ export default abstract class AbstractController<T extends AbstractService> {
 		return this._router;
 	}
 
+	public get cache(): CacheType {
```

Comment:
This shouldn't be here? Looks like it should be in BlocksController.

Reply:
So the first time around I actually had the cache getter, setter, and object inside of `BlocksController`, but `BlocksExtrinsicController` also needs access to the cache so I decided I would put it in AbstractController instead.

Reply:
Fixed!